### ISSUE TITLE
Which trained model was used for validation

### ISSUE THREAD

**Issue by PatrickStarMS** [ductuantruong/enskd#3]
hello,thanks for the code.
1. can you tell me that you are using avg_model.pt for training?
 2.And why did the teacher print out his accuracy on the training set when he was training, and the result was about 87.
3.When I trained, I put the following
 ”configs['scheduler_args']['scale_ratio'] = 1.0 * world_size * configs[
            The 'dataloader_args']['batch_size']/64“code is commented out, and the log shows that the acc on the training set is higher than what you provided, but at the time of validation, it is very poor, will there be a problem here？
